txtdirect.go

```diff
@@ -160,6 +160,16 @@ func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
 		return nil
 	}
 
+	// Add referer header
+	if rec.Ref && r.Header.Get("Referer") == "" {
```

Comment:
In which case is the Referer header already set?

Reply:
Maybe when the user sends the request from a search engine or something like that? Added this condition based on your comment on the issue.
https://github.com/txtdirect/txtdirect/issues/273#issuecomment-510471223

Reply:
Ah yeah. Playing nice with other referers. Thanks for pointing that out. /lgtm.